app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/PublicationsFragment.java

```diff
@@ -39,6 +102,37 @@ public void initializeUI() {
     @Override
     public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         inflater.inflate(R.menu.menu, menu);
+
+        final MenuItem item = menu.findItem(R.id.menu_search);
+        final SearchView searchView = (SearchView) MenuItemCompat.getActionView(item);
+        searchView.setQueryHint("Search by Title");
+        searchView.setOnQueryTextListener(this);
+        searchView.setOnCloseListener(new SearchView.OnCloseListener() {
+            @Override
+            public boolean onClose() {
+                return false;
+            }
+        });
+
         super.onCreateOptionsMenu(menu, inflater);
     }
-}
+
+    @Override
+    public boolean onQueryTextSubmit(String query) {
+        return false;
+    }
+
+    @Override
+    public boolean onQueryTextChange(String query) {
+        if (query.length() <= 0) {
+            mListView.setAdapter(mTestAdapter);
+        }
+
+        mTestAdapter.getFilter().filter(query);
+        mListView.setAdapter(mTestAdapter);
+        
+        //mRecyclerView.scrollToPosition(0);
+        mListView.smoothScrollToPosition(0);
```

Comment:
What is 0 ?  A magic number? Just extract this as a constant : `public static final int DEFAULT_POSITION = 0` :)